python/mxnet/numpy/multiarray.py

```diff
@@ -3591,6 +3589,55 @@ def remainder(x1, x2, out=None, **kwargs):
     """
     return _mx_nd_np.remainder(x1, x2, out=out)
 
+@set_module('mxnet.numpy')
+@wrap_np_binary_func
+def pow(x1, x2):
```

Comment:
We can use assignment to make `pow` the alias of `power` and add notes to inform the user that `pow` is not a official NumPy operator but a standard API. 

Example:
```
pow = power
pow.__doc__ = ```
    ...
    Notes
    -----
    `pow` is an alias for `power`. It is a standard API in https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html#pow-x1-x2 instead of an official NumPy operator. 

    >>> np.pow is np.power
    True
    ...
```

We can apply this to other operators in this PR. What do you think.